doc/umock_c.md

```diff
@@ -1328,3 +1329,54 @@ If `ENABLE_MOCKS` is not defined, `IMPLEMENT_MOCKABLE_FUNCTION` shall expand to
 ```
 
 `UMOCK_REAL` shall produce the name of the real function generated by umock.
+
+### umock_c lock factory
+
+```c
+int umock_c_init_with_lock_factory(ON_UMOCK_C_ERROR on_umock_c_error, UMOCK_C_LOCK_FACTORY_CREATE_LOCK_FUNC lock_factory_create_lock, void* lock_factory_create_lock_params);
+```
+
+There are scenarios where mocking is done in multi threaded environments.
```

Comment:
>mocking is done [](start = 26, length = 15)

"mocked calls are made" (sounds a bit more clear to me in terms of what this solves, multi-threaded calls to mock functions)

Reply:
Added

---
In reply to: [530651263](https://github.com/Azure/umock-c/pull/177#discussion_r530651263) [](ancestors = 530651263)